### ISSUE TITLE
Streams not loading when a user is logged out

### ISSUE THREAD

**Issue by Dman757** [MemeLabs/Rustla2#334]
*Issue:* When I am logged out and click on a stream from the home page the iframe doesn't load unless the stream page is refreshed.

For some context I asked about this bug in the #feedback channel on discord and started doing some digging: https://discordapp.com/channels/464209732291264513/464211660400361492/779870252858081351

### *Issue Report*
This is the dom node when navigating to the page:
![chrome_Rtgotj8rsq](https://user-images.githubusercontent.com/4275720/99898370-cee6b300-2c55-11eb-8835-015ffb2692b9.png)

After a refresh (or being logged in) it is populated
![chrome_3S68NtXruR](https://user-images.githubusercontent.com/4275720/99898377-e2921980-2c55-11eb-8ba8-e191ab995d76.png)

### Some Investigation
I investigated the issue a little bit using the chrome and react devtools in the browser.

On the home page I get this type error as well as a few 401's attempting to fetch profile info:
![chrome_g4uIZNy4T9](https://user-images.githubusercontent.com/4275720/99898384-f2116280-2c55-11eb-8c66-8a433874a131.png)

Prettifying the code from the type error it appears to be something with profile login attempt failure:
![chrome_i0WVe8UcLg](https://user-images.githubusercontent.com/4275720/99898389-02294200-2c56-11eb-97d6-cb72e3a0c22c.png)

Inspecting the react tree with react-devtools it appears that a loading variable `isFetchingProfile` is never set to `false` 
![chrome_VbqyyFs0ih](https://user-images.githubusercontent.com/4275720/99898455-8085e400-2c56-11eb-8520-4eaafd8b4b24.png)

I did a quick github search of this repo for `isFetchingProfile` it would appear that this `recompose` `branch()` function would cause `renderNothing` does what the name implies.
https://github.com/MemeLabs/Rustla2/blob/7e636f78ad8ce0a4e10ad54a3fe7a4654ebebc71/src/components/Stream.jsx#L108

***TL;DR*** It looks like there's an error thrown for non-authenticated users that causes `isFetchingProfile` to always be `true` thus causing [Stream.jsx](https://github.com/MemeLabs/Rustla2/blob/7e636f78ad8ce0a4e10ad54a3fe7a4654ebebc71/src/components/Stream.jsx#L108) to never return anything.

Also I guess I should note I haven't run the build locally. Just hope this helps someone resolve the issue.
